source/_integrations/integration.markdown

```diff
@@ -93,7 +93,7 @@ method:
   default: trapezoidal
 {% endconfiguration %}
 
-In case you have an appliance which produces spikey consumption (like an on/off electrical boiler) you should opt for the `left` method to get accurate readings.
+In case you expect that your source sensor will provide several subsequent values that are equal, you should opt for the `left` method to get accurate readings. Due to limitations in Home Assistant, the `right` and `trapezoidal` methods will [provide incorrect values](https://github.com/home-assistant/core/issues/74434) in such cases.
```

Comment:
We don't document issues in our documentation; as it ends up sticking there for a long time (even after the issues are solved). Thus are hard to maintain.

Reply:
I understand your argumentation and adapted the text.
What is a good way then to inform users about the reason why they should opt for the 'left' method in these cases? There are probably many tech-savvy users who are rather less happy with just being told to use a certain integration method in certain cases, and would like to understand why the documentation is asking them to select the 'left' method.

Reply:
> What is a good way then to inform users about the reason why they should opt for the 'left' method in these cases?

That is a good question. I think best is to fix things 😬 Take away these limitations? Maybe you can help out with that!